fastlane/metadata/fr-FR/subtitle.txt

```diff
@@ -1 +1 @@
-Concevez un site, créez un blog
\ No newline at end of file
+Concevez un site créez un blog
```

Comment:
This is a brilliant fix for the 1 extra character issue with the `fr-FR` translation! I wish I thought of that 🤦‍♂️